libs/ui/src/FileList/FileListItem.tsx

```diff
@@ -1,61 +1,62 @@
 import { Button } from '@amsterdam/design-system-react/dist/Button'
-import useIsAfterBreakpoint from '@amsterdam/design-system-react/dist/common/useIsAfterBreakpoint'
-import { ErrorMessage } from '@amsterdam/design-system-react/dist/ErrorMessage'
+import { Icon } from '@amsterdam/design-system-react/dist/Icon'
 import { Image } from '@amsterdam/design-system-react/dist/Image'
-import { clsx } from 'clsx'
+import { WarningIcon } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
 import { HTMLAttributes, useMemo } from 'react'
 
-import { formatFileSize } from './formatFileSize'
-
 import styles from './FileList.module.css'
 
 // Although a description list (<dl>, <dt> and <dd>) would be more semantically correct than an unordered list (<ul> and <li>),
 // an unordered list with list items is used here because NVDA currently (16-9-2025) reads the number of items in a description list incorrectly.
 
 export type FileListItemProps = HTMLAttributes<HTMLLIElement> & {
   deleteButtonId: string
-  deleteButtonLabel?: string
   errorMessage?: string
   file: File
   onDelete?: () => void
+  progress: number
+  status: 'pending' | 'uploading' | 'success' | 'error'
 }
 
-export const FileListItem = ({
-  deleteButtonId,
-  deleteButtonLabel = 'Verwijder',
-  errorMessage,
-  file,
-  onDelete,
-}: FileListItemProps) => {
+export const FileListItem = ({ deleteButtonId, errorMessage, file, onDelete, progress, status }: FileListItemProps) => {
   // Memoize the creation of an object url from the file,
   // to prevent it from creating a new one on every render.
   const imageUrl = useMemo(() => URL.createObjectURL(file), [file])
 
+  const actionButtonLabel = status === 'success' || status === 'error' ? 'Verwijderen' : 'Annuleren'
```

Comment:
We generally don't want to hardcode these texts.

Reply:
Changed it, but thought it was a bit of over-engineering to add 4 labels to the props that you probably never gonna change.